tests/test_electrolyzer.py

```diff
@@ -200,9 +217,14 @@ def test_electrolyzer_minimum_power(self):
         LINEARIZED_THREE_LINES_WEAK_INEQUALITY.
 
         Checks:
-        - Input power to the electrolyzer is 0
-        - Output gas is 0
-        - Electrolyzer is switched off
+        - Input power to the electrolyzer is 0 when available wind power is 49MW
```

Comment:
Add remark that it should be zero because threshold for electrolyzer is 50MW

Reply:
Done